### PR TITLE
[SD-4397]-(fix) - Action menu should refresh after a highlight is deleted from highlights section

### PR DESCRIPTION
**Pull request by sjunaid**

### CODE REVIEW

File: scripts/superdesk-highlights/highlights.js [hunk 1]
```diff
@@ -418,6 +418,7 @@
             .then(function() {
                 api.highlights.remove(config).then(function() {
                     _.remove($scope.configurations._items, config);
+                    highlightsService.clearCache();
```
Comment (petrjasek) [inline scripts/superdesk-highlights/highlights.js]: imo this should be part of some `highlightsService.removeConfig` function

**Comment by mdhaman** [comment]
@sjunaid squash and change the jira ticket to caps i.e. SD-4397.

**Comment by petrjasek** [comment]
:+1:

**Comment by petrjasek** [comment]
and :+1: for this e2e error fix

**Pull request merged**